modules/pam_faillock/tests/faillock.conf.in

```diff
@@ -0,0 +1,62 @@
+# Configuration for locking the user after multiple failed
+# authentication attempts.
```

Comment:
It will be used in tst-pam_faillock-retval.c. 
* Testing if loading of a special config file is working at all.
* Decreasing fail attempts and timeout value.

While building faillock.conf will be generated from faillock.conf.in automatically.
Is this way wrong or is there any better one ?